paparazzi-preview-processor/src/test/java/app/cash/paparazzi/preview/processor/PreviewProcessorProviderTest.kt

```diff
@@ -289,5 +298,19 @@ class PreviewProcessorProviderTest {
         annotation class Paparazzi
         """.trimIndent()
       )
+
+    private val PAPARAZZI_PREVIEW_DATA_SOURCE = SourceFile.kotlin(
+      "PaparazziPreviewData.kt",
+      """
+        package app.cash.paparazzi.annotations
+
+        import androidx.compose.runtime.Composable
+
+        data class PaparazziPreviewData(
+          val snapshotName: String,
+          val composable: @Composable () -> Unit
+        )
+        """.trimIndent()
+    )
```

Comment:
Not sure how codegen tests were passing when the generated code would fail to compile without this class.

Good news is now it works as expected 🎉